app/portainer/views/stacks/create/createStackController.js

```diff
@@ -14,7 +14,8 @@ angular
     FormValidator,
     ResourceControlService,
     FormHelper,
-    CustomTemplateService
+    CustomTemplateService,
+    EndpointProvider
```

Comment:
```suggestion
    endpoint
```

it's possible just to use `endpoint` here